diff --git a/src/twisted/spread/test/test_pbfailure.py b/src/twisted/spread/test/test_pbfailure.py
index 57a5716..12aee4a 100644
--- a/src/twisted/spread/test/test_pbfailure.py
+++ b/src/twisted/spread/test/test_pbfailure.py
@@ -306,36 +306,6 @@ class PBFailureTests(PBConnTestCase):
 
         return self._testImpl("unknownError", 4310, failureUnknown)
 
-    def test_securityFailure(self):
-        """
-        Test that even if an exception is not explicitly jellyable (by being
-        a L{pb.Jellyable} subclass), as long as it is an L{pb.Error}
-        subclass it receives the same special treatment.
-        """
-
-        def failureSecurity(fail):
-            fail.trap(SecurityError)
-            self.assertNotIsInstance(fail.type, str)
-            self.assertIsInstance(fail.value, fail.type)
-            return 4300
-
-        return self._testImpl("security", 4300, failureSecurity)
-
-    def test_deferredSecurity(self):
-        """
-        Test that a Deferred which fails with a L{pb.Error} which is not
-        also a L{pb.Jellyable} is treated in the same way as a synchronously
-        raised exception of the same type.
-        """
-
-        def failureDeferredSecurity(fail):
-            fail.trap(SecurityError)
-            self.assertNotIsInstance(fail.type, str)
-            self.assertIsInstance(fail.value, fail.type)
-            return 43000
-
-        return self._testImpl("deferredSecurity", 43000, failureDeferredSecurity)
-
     def test_noSuchMethodFailure(self):
         """
         Test that attempting to call a method which is not defined correctly
